src/DynamoApplications/StartupUtils.cs

```diff
@@ -281,6 +297,20 @@ public static DynamoModel MakeModel(bool CLImode, string asmPath = "", HostAnaly
             return model;
         }
 
+        /// <summary>
+        /// It returns an IPathResolver based on the mode and some locations
+        /// </summary>
+        /// <param name="CLImode">CLI mode starts the model in test mode and uses a seperate path resolver.</param>
+        /// <param name="preloaderLocation">Path to be used by PathResolver for preLoaderLocation</param>
+        /// <param name="userDataFolder">Path to be used by PathResolver for UserDataFolder</param>
+        /// <param name="commonDataFolder">Path to be used by PathResolver for CommonDataFolder</param>
+        /// <returns></returns>
+        private static IPathResolver CreateIPathResolver(bool CLImode, string preloaderLocation, string userDataFolder, string commonDataFolder)
```

Comment:
Better call this  `CreateIPathResolver` because `IPathResolver` is the name of the interface, not class